frontend/src/components/articles/ArticleItem.vue

```diff
@@ -7,17 +7,28 @@ import InputNumber from "primevue/inputnumber";
 import InputText from "primevue/inputtext";
 import Textarea from "primevue/textarea";
 import { ref } from "vue";
+import { useRouter } from "vue-router";
 
+import { deleteArticle, updateArticle } from "@/backendClient";
 import { ArticleKind } from "@/global/types/appTypes";
 import type { Article } from "@/global/types/entities";
 
 const props = defineProps<{
   article: Article;
 }>();
+const router = useRouter();
 
 const articlesType = Object.values(ArticleKind);
 
 const editableArticle = ref(props.article);
+
+const onUpdateArticle = async () => {
+  await updateArticle(`${editableArticle.value.id}`, editableArticle.value);
+};
+const onDeleteArticle = async () => {
+  await deleteArticle(`${editableArticle.value.id}`);
+  router.go(0);
+};
```

Comment:
I feel like we need a confirmation modal or something to not delete items by miss-clicking 😅